src/app/pages/fund/types.ts

```diff
@@ -0,0 +1,4 @@
+export enum CurrencySymbol {
+  btc = 'btc',
+  stx = 'stx',
+}
```

Comment:
Even though this is only used in fund page, I see this as a concern that belongs with the other currency code in `src/shared/models/currencies.model.ts`